pkgs/top-level/python-packages.nix

```diff
@@ -623,12 +623,12 @@ in {
   };
 
   atomicwrites = buildPythonPackage rec {
-    version = "0.1.9";
+    version = "1.1.5";
```

Comment:
As `atomicwrites` was moved to a dedicated directory in #38439, it probably makes sense to rebase on staging.